vendor/github.com/gogo/protobuf/plugin/marshalto/marshalto.go

```diff
@@ -122,7 +122,7 @@ This is followed by calling the MarshalTo method which requires a preallocated b
 The MarshalTo method allows a user to rather preallocated a reusable buffer.
 
 The Size method is generated using the size plugin and the gogoproto.sizer, gogoproto.sizer_all extensions.
-The user can also using the generated Size method to check that his reusable buffer is still big enough.
+The user can also using the generated Size method to check that their reusable buffer is still big enough.
```

Comment:
I'm not sure that modifying a third-party code is a good ideas as it could create difficulties in a future when we will update such dependencies.

@kennethgillen Have you already submit PRs for gogo/protobuf and vishvananda/netlink repos? I'd suggest you to do that.

Reply:
@php-coder - right: didn't realise it was external, just ran through the repo with `find`.

Thanks for the pointer.